client/src/components/scans/scanJobsList.js

```diff
@@ -0,0 +1,132 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Dropdown, EmptyState, Grid, Icon, MenuItem, Modal } from 'patternfly-react';
+import * as moment from 'moment/moment';
+
+class ScanJobsList extends React.Component {
+  renderJob(job) {
+    const { onSummaryDownload, onDetailedDownload } = this.props;
+    let scanDescription = '';
+
+    let icon = null;
+    let scanTime = _.get(job, 'end_time');
+
+    switch (job.status) {
+      case 'completed':
+        scanDescription = 'Complete';
+        icon = <Icon className="scan-job-status-icon" type="pf" name="ok" />;
+        break;
+      case 'failed':
+        scanDescription = 'Failed';
+        icon = <Icon className="scan-job-status-icon" type="pf" name="error-circle-o" />;
+        break;
+      case 'canceled':
+        scanDescription = 'Canceled';
+        icon = <Icon className="scan-job-status-icon" type="pf" name="error-circle-o" />;
+        break;
+      case 'created':
+        scanDescription = 'Created';
+        icon = <Icon className="scan-job-status-icon invisible" type="fa" name="spinner" />;
+        break;
+      case 'pending':
+        scanDescription = 'Pending';
+        icon = <Icon className="scan-job-status-icon invisible" type="fa" name="spinner" />;
+        scanTime = _.get(job, 'start_time');
+        break;
+      case 'running':
+        scanDescription = 'In Progress';
+        icon = <Icon className="scan-job-status-icon fa-spin" type="fa" name="spinner" />;
+        scanTime = _.get(job, 'start_time');
+        break;
+      case 'paused':
+        scanDescription = 'Paused';
+        icon = <Icon className="scan-job-status-icon" type="pf" name="warning-triangle-o" />;
+        break;
+      default:
+        return null;
+    }
```

Comment:
Note: the switch might be an opportunity for some clean up next round, little repetition